### PR TITLE
feat: [FFM-4523] FF Overview - More info before the FF module is started

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

Redesign of the FF Overview to provide more information as well as added 'Invite Developer'button.

#### Screenshots

Ship more features and reduce risk with Feature Flags!

h Feature

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by chrisblakely01 — APPROVED** [review]

File: src/modules/75-cf/pages/home/CFTrialTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+  PENDING = 'PENDING'
+}
+
+export interface CardSectionProps {
+  title: ReactNode
+  listItems: ReactNode[]
+}
+
+const CardSection: FC<CardSectionProps> = ({ title, listItems }) => (
+  <section className={css.forDevs}>
```
Comment (Kira-Jennings) [inline src/modules/75-cf/pages/home/CFTrialTemplate.tsx:67]: This classname can be avoided by using a Layout component e.g.
`<Layout.Vertical flex spacing="small">`

File: src/modules/75-cf/pages/home/CFTrialTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+            <String
+              key={getString('cf.cfTrialHomePage.forDevelopers.realTime')}
+              stringID="cf.cfTrialHomePage.forDevelopers.realTime"
+              useRichText
+            />
+          ]}
+        />
+
+        {/* Divider */}
+        <div className={css.divider} />
```
Comment (Kira-Jennings) [inline src/modules/75-cf/pages/home/CFTrialTemplate.tsx:156]: There is an existing component called Divider (just fyi)

File: src/modules/75-cf/pages/home/CFTrialTemplate.tsx [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+          variation={ButtonVariation.PRIMARY}
+          text={startBtn.description}
+          onClick={startBtn.onClick ? startBtn.onClick : handleStartButtonClick}
+          disabled={loading}
+        />
+      </article>
+
+      {/* Don't Code Panel */}
+      <article className={css.dontCodeArticle}>
+        <section className={css.dontCode}>
```
Comment (Kira-Jennings) [inline src/modules/75-cf/pages/home/CFTrialTemplate.tsx:181]: display: flex, flexDirection and width can be set on the element here if you use a <Layout> component
e.g. <Layout.Vertical width="450px">

File: src/modules/75-cf/pages/home/CFTrialTemplate.module.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.body {
+  padding-top: 90px !important;
+  padding-left: 80px !important;
+  height: 100vh;
```
Comment (Kira-Jennings) [inline src/modules/75-cf/pages/home/CFTrialTemplate.module.scss:11]: Can the first three items here be set on the element instead?

**Review by Kira-Jennings — DISMISSED** [review]

File: src/modules/75-cf/pages/home/CFTrialTemplate.tsx (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/home/CFTrialTemplate.tsx:67]: Thanks, as discussed will address in refactor PR

**Review by Kira-Jennings — DISMISSED** [review]

File: src/modules/75-cf/pages/home/CFTrialTemplate.tsx (hunk 2 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/home/CFTrialTemplate.tsx:156]: Thanks, as discussed will address in refactor PR

File: src/modules/75-cf/pages/home/CFTrialTemplate.tsx (hunk 3 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/home/CFTrialTemplate.tsx:181]: Thanks, as discussed will address in refactor PR

File: src/modules/75-cf/pages/home/CFTrialTemplate.module.scss (hunk 4 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/home/CFTrialTemplate.module.scss:11]: Thanks, as discussed will address in refactor PR

**Review by devinder-harness — APPROVED** [review]

**Review by Kira-Jennings — APPROVED** [review]

**Review by chrisblakely01 — APPROVED** [review]

**Review by devinder-harness — APPROVED** [review]

File: src/modules/75-cf/pages/home/__test__/CFTrialHomePage.test.tsx [hunk 5]
```diff
@@ -1,53 +1,130 @@
...
+
+    jest.spyOn(cdServices, 'useStartFreeLicense').mockReturnValue({
+      mutate: handleStartButtonClick,
+      loading: false
+    } as any)
+
+    renderComponent({ FREE_PLAN_ENABLED: true })
+    expect(handleStartButtonClick).not.toHaveBeenCalled()
+    userEvent.click(screen.getByText('cf.cfTrialHomePage.startFreePlanBtn'))
+    await waitFor(() => expect(handleStartButtonClick).toHaveBeenCalled())
```
Comment (knagurski) [inline src/modules/75-cf/pages/home/__test__/CFTrialHomePage.test.tsx:114]: Not a problem, but try to keep in mind setup/action/assert and separate them with some white space. A few line breaks make a huge difference to the scanability of a test.

**Review by knagurski — COMMENTED** [review]
Some issues to address, but we can do so in the follow on PR 👍🏻

**Pull request merged**